lib/src/sv_sign.c

```diff
@@ -860,6 +862,16 @@ SignedVideoReturnCode
 signed_video_set_private_key(signed_video_t *self, const char *private_key, size_t private_key_size)
 {
   if (!self || !private_key || private_key_size == 0) return SV_INVALID_PARAMETER;
+#if defined(SIGNED_VIDEO_DEBUG)
+#endif
+  // If ONVIF is available, call its function and map the return code
+  if (self->onvif) {
```

Comment:
Note that the same action inside this if-statement is also done in `sv_vendor_axis_communications_set_attestation_report(...)`. So, isn't it possible to create a function that does this and call it from both places?